src/engine/controls/bpmcontrol.cpp

```diff
@@ -539,7 +539,7 @@ bool BpmControl::getBeatContext(const mixxx::BeatsPointer& pBeats,
 
     double dPrevBeat;
     double dNextBeat;
-    if (!pBeats->findPrevNextBeats(dPosition, &dPrevBeat, &dNextBeat)) {
+    if (!pBeats->findPrevNextBeats(dPosition, &dPrevBeat, &dNextBeat, false)) {
```

Comment:
We can use her true and remove undoing the fix in getBeatContextNoLookup(). 
This is actually a bug in case of a beat map where the beats can be differently.

Reply:
I changed this, but I don't know how to verify this.